src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -48,6 +53,36 @@ subroutine read_geogrid(fname, rarray, nx, ny, nz, isigned, endian, &
     end subroutine read_geogrid
  end interface
 
+ ! Abstract interface for determining if the cell, iCell, needs to be added to the stack
+ ! for processing. If an interface returns .true., it will indicate to the calling code
+ ! that the cell has not recived any mappings and needs to be processed. Returning .false.
+ ! will indicate that the cell has recived mappings and does not need to be procssed.
+ abstract interface
+    function interp_criteria_function(mgr, iCell)
+        import mpas_geotile_mgr_type
```

Comment:
Would it be better to change the `import` here (and also in the `interp_accumulation_function` interface) to a `use` statement?